##
# @File: Neighbors.srv
#
# @brief
# Input: string of the robot_id (robot namespace) whose neighbors are being 
# requested
# Output: Neighbors message
##

string robot_id
---
mv_msgs/Neighbors neighbors